building-with-containers/azure-go/answers/lab4-hints.md

```diff
@@ -0,0 +1,6 @@
+# Hints
+
+- Where have you hunted for the secret?
```

Comment:
Could we replace 'hunted' with 'searched' as a non-violent word?

Reply:
Good call!